website/www/site/content/en/documentation/pipelines/test-your-pipeline.md

```diff
@@ -284,3 +345,32 @@ public class WordCountTest {
     }
 }
 {{< /highlight >}}
+
+{{< highlight py >}}
+class WordCountTest(unittest.TestCase):
+
+  # Our input data, which will make up the initial PCollection.
+  WORDS = [
```

Comment:
Out of curiosity: why are the list names in all caps? I don't see anything in the PEP8 against it, but it's inconsistent with most of our code samples.

Reply:
Mostly it's for consistency with the Java code and context, and it is a convention to use for globals even in Python.